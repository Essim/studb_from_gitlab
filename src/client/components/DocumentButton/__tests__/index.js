import React from 'react';
import {DocumentButton} from '../';
import { shallowToJson } from 'enzyme-to-json';

const mockButton = "like",
    mockDocument_cuid="mockCUID",
    mockOpinions=["mockOpinion1","mockOpinion2","mockOpinion3"];

describe('DocumentButton', () => {
    it('should render correctly', () => {
        const output = shallow(<DocumentButton button={mockButton} document_cuid={mockDocument_cuid} opinions={mockOpinions}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
