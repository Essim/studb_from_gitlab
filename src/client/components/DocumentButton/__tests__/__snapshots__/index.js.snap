// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentButton should render correctly 1`] = `
<Connect(LikeButton)
  document_cuid="mockCUID"
  key="like_mockCUID"
  opinions={
    Array [
      "mockOpinion1",
      "mockOpinion2",
      "mockOpinion3",
    ]
  }
/>
`;
